Gemfile

```diff
@@ -1,5 +1,20 @@
-gem 'sinatra'
+source 'https://rubygems.org'
+
+gem 'rack'
+gem 'sinatra', require: 'sinatra/base'
+gem 'sinatra-contrib'
 gem 'figaro'
 gem 'faraday'
 gem 'pry'
-gem 'rspec'
+gem 'json'
+gem 'fast_jsonapi'
+gem 'shotgun'
```

Comment:
**Note for myself:** The shotgun command starts one of Rack's supported servers (e.g., mongrel, thin, webrick) and listens for requests but does not load any part of the actual application. Each time a request is received, it forks, loads the application in the child process, processes the request, and exits the child process. The result is clean, application-wide reloading of all source files and templates on each request. _Shotgun only works on Mac and Linux_

https://learn.co/lessons/sinatra-shotgun-server